### ISSUE TITLE
Closes on launch with windows

### ISSUE THREAD

**Issue by SquidRollr** [Shugabuga/Splatoon2RichPresence#1]
Crashes on launch with windows build: 17134 (10.0.17134)
Python has been reinstalled but no fix.

**Comment by Shugabuga** [comment]
I mainly use macOS but will try to investigate. Thanks for letting me know of the issue!

**Comment by Shugabuga** [comment]
@SquidRollr Do you have an error log so I can better diagnose the problem? Seems that #2 is also describing a similar issue.

**Comment by SquidRollr** [comment]
Where's an error log?

**Comment by Shugabuga** [comment]
@SquidRollr Does the Python instance have any traceback? It should spew a bunch of text when it fails.
I need that.

**Comment by SquidRollr** [comment]
import time, tkinter, requests
ModuleNotFoundError: No module named 'requests'
line 7, in <module>

**Comment by SquidRollr** [comment]
Traceback (most recent call last):
  File "C:\Users\Owner\Downloads\Splatoon2RichPresence-master\Splatoon2RichPresence-master\launch.py", line 7, in <module>
    import time, tkinter, requests
ModuleNotFoundError: No module named 'requests'

**Comment by Shugabuga** [comment]
Do you have `requests` installed? `pip` should come with it, and `pip` should be shipped with `python3`.

**Comment by SquidRollr** [comment]
Yeah, I've even reinstalled them.

**Comment by hagretek** [comment]
Once the code has been run, the terminal **closes automatically**. Maybe that is the problem?

You could try to fix it with this:
```python
while running == True:
    dontclose(i-dont-really-know-this-is-just-an-example)
```
Or with something like that.

&nbsp;

### Edit:
wow this question is old

**Comment by Shugabuga** [comment]
@HagRetek Yeah. I need to find a way to get a hold of a Windows machine so I can try to fix this.

**Comment by hagretek** [comment]
@Shugabuga Just download Oracle VirtualBox. Microsoft looks at Windows 10 as a service now, so there are enough virus-free Windows 10 mirrors and maybe one official from Microsoft.

**Comment by Shugabuga** [comment]
@HagRetek I don't know why I didn't think of that, given that I have an older Win10 install disk and VirtualBox. I guess my tiny SSD dissuaded me from going that route.

Thanks for the idea!
